python/tests/mode_coeffs.py

```diff
@@ -117,8 +117,8 @@ def test_modes(self):
         eval_point = mp.Vector3(0.7, -0.2, 0.3)
         ex_at_eval_point = emdata.amplitude(eval_point, mp.Ex)
         hz_at_eval_point = emdata.amplitude(eval_point, mp.Hz)
-        self.assertAlmostEqual(ex_at_eval_point, 0.45358518109307083+0.5335421986481814j)
-        self.assertAlmostEqual(hz_at_eval_point, 3.717865162096829-3.1592989829386298j)
+        self.assertAlmostEqual(ex_at_eval_point, 0.4887779638178009+0.48424014532428294j)
+        self.assertAlmostEqual(hz_at_eval_point, 3.4249236584603495-3.455974863884166j)
```

Comment:
Is this discretization error?

That is, do the old code and the new code converge to one another (at a reasonable rate) as you increase the resolution?

Reply:
Yes, this is indeed a discretization error: the values from the two versions of the code are converging to each other with increasing resolution.

Reply:
If you output epsilon in both cases, how do they differ?

Reply:
It would also be nice to plot the error vs resolution for both old and new codes (compared to a high-resolution run, I guess).

Reply:
When these values are compared as the absolute-value squared, the differences are found to be minor.

**pre libctl 53**
abs(0.45358518109307083+0.5335421986481814j)**2 --> 0.4904067942455693
abs(3.717865162096829-3.1592989829386298j)**2 --> 23.803691427130342

**post libctl 53**
abs(0.4887779638178009+0.48424014532428294j)**2 --> 0.47339241625755824
abs(3.4249236584603495-3.455974863884166j)**2 --> 23.673864326080604

The relative differences in these two sets of values are 3.6% and 0.5%.